charts/tidb-cluster/templates/monitor-configmap.yaml

```diff
@@ -19,7 +19,7 @@ data:
 {{ tuple "config/_grafana-datasource.tpl" . | include "helm-toolkit.utils.template" | indent 4 }}
   dashboard-config: |-
 {{ tuple "config/_grafana-dashboard.tpl" . | include "helm-toolkit.utils.template" | indent 4 }}
-  grafana-config: |-
-{{ tuple "config/_grafana-config.tpl" . | include "helm-toolkit.utils.template" | indent 4 }}
+  grafana-prestart: |
+{{- include "setup" . | indent 4 }}
```

Comment:
Why not use the same way with other [configmaps](https://github.com/pingcap/tidb-operator/blob/master/charts/tidb-cluster/templates/pd-configmap.yaml#L13) that include scripts? The scripts you define in this way cannot be executed separately.